File: Backend/SimpleKanbanBoards.DataAccess/Models/BoardColumn.cs
using System;
using System.Collections.Generic;

namespace SimpleKanbanBoards.DataAccess.Models;

public partial class BoardColumn
{
    public int IdBoardColumn { get; set; }

    public int? IdBoard { get; set; }

    public string? BoardColumnName { get; set; }

    public int? ColumnPosition { get; set; }

    public int? WipLimit { get; set; }

    public bool? IsEntry { get; set; }

    public bool? IsDone { get; set; }

    public virtual Board? IdBoardNavigation { get; set; }

    public virtual ICollection<Task> Tasks { get; set; } = new List<Task>();
}

File: Frontend/simple-kanbanboards-client/src/boards/types/boardColumn.ts
export interface BoardColumnModel {
  id: number;
  name: string;
  position: number;
  wipLimit: number;
  isEntry: boolean;
  isDone: boolean;
  idBoard: number;
}

export interface CreateBoardColumnModel {
  name: string;
  position: number;
  wipLimit: number;
  isEntry: boolean;
  isDone: boolean;
  idBoard: number;
}

export interface UpdateBoardColumnModel {
  id: number;
  name: string;
  position: number;
  wipLimit: number;
  isEntry: boolean;
  isDone: boolean;
}
